osquery/tables/events/darwin/openbsm_events.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <string>
+#include <vector>
+
+#include <bsm/audit.h>
+#include <bsm/libbsm.h>
+
+#include <osquery/config.h>
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/events/darwin/openbsm.h"
+#include "osquery/tables/events/event_utils.h"
+
+namespace osquery {
+
+// OpenBSM token parsers
+static inline void OpenBSM_AUT_SUBJECT32_EX(Row& r, const tokenstr_t& tok) {
```

Comment:
I took OpenBSM_ then suffixed it with the exact token name to make it easy to find them in future while giving a sane naming scheme. If you want to change the scheme though that's fine with me.